webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -75,23 +73,28 @@ def search_spotify_artist(artist_name):
     response = requests.get(url, headers=spotify_authenticate(), params=params)
     artists_results = json.loads(response.text)
 
+    #TODO Return a list of artists for a dropdown menu
```

Comment:
this you need help with right? can you detail what you try to achieve here? maybe we need to break it down into multiple steps?